src/containers/MasonryLayout.jsx

```diff
@@ -55,13 +55,19 @@ class MasonryLayout extends Component {
    * @return {object} JSX for MasonryLayout component
    */
   render() {
+    const { items } = this.state;
     return (
       <div className='masonry'>
-        <div className='masonry-layout'>
-          {
-            this.createItems(this.state.items)
-          }
-        </div>
+        {
+          items.length ?
+            <div className='masonry-layout'>
+              {
+                this.createItems(items)
+              }
+            </div>
+            :
+            <h2 className='items-unavailable-message'>There are no activities at the moment</h2>
```

Comment:
Is it possible we make the error messages unique? There is a component, `ErrorMessage` component we use for displaying such messages. I would advise you make use of it.

Reply:
@Chris-Maina All done